contrib/labstack/echo.v4/echotrace.go

```diff
@@ -47,7 +49,15 @@ func Middleware(opts ...Option) echo.MiddlewareFunc {
 			defer span.Finish()
 
 			// pass the span through the request context
-			c.SetRequest(request.WithContext(ctx))
+			req := request.WithContext(ctx)
+			c.SetRequest(req)
+
+			if appsec.Enabled() {
+				op := httpinstr.StartOperation(httpinstr.MakeHandlerOperationArgs(req, span), nil)
```

Comment:
I'm starting to feel quite strongly about renaming this package to `httpsec`. Reading code in the tracer now becomes ambiguous when seeing things like `httpinstr.WrapHandler`. It makes one think "HTTP instrumentation", which in this context, is tracing. We should be clear. I think `httpsec` is a great name ("HTTP Security") and we should probably use this `*sec` suffix on all AppSec integrations, just like we use the `*trace` suffix on all tracing integrations (granted its being renamed at import time but too late to change that).

I have suggested this before. Is there any reason you are against it?

Reply:
It's still too early to say this is completely appsec-specific for me. I see some functions currently not covered by spans, like the request body reads, that appsec will need to cover it soon with some new `httpinstr` operations and this will be the moment where the tracer could also create a span as an instrumentation gateway operation callback - because this will be our first time a tracer integration doesn't exist for that. In the base architecture idea, this instrumentation gateway was meant to be used by anyone, this is why I still would like to wait bit and see if it ends up only applied to appsec.

Reply:
Right. But it is part of the `appsec` package, so it is fully AppSec related for now. Would you mind doing this change? I think it offers a much better experience. If we ever end up finding that we need to change the name, it is of course perfectly fine to do so. For now, nothing else is using this for any other purposes than AppSec, and it clashes in concept with HTTP instrumentation of the tracer.